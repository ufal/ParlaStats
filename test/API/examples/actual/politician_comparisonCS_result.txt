[{'person.person_id': 'AdamKalous.1979', 'persName.forename': 'Adam', 'persName.surname': 'Kalous', 'total_speeches': 78}, {'person.person_id': 'AdamRykala.1986', 'persName.forename': 'Adam', 'persName.surname': 'Rykala', 'total_speeches': 29}]

+------------------+-------------------+------------------+----------------+
| person.person_id | persName.forename | persName.surname | total_speeches |
+------------------+-------------------+------------------+----------------+
| AdamKalous.1979  |        Adam       |      Kalous      |       78       |
| AdamRykala.1986  |        Adam       |      Rykala      |       29       |
+------------------+-------------------+------------------+----------------+
