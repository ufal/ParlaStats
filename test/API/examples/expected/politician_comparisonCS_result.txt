[{'person.person_id': 'AndrejBabis.1954', 'persName.forename': 'Andrej', 'persName.surname': 'Babiš', 'total_speeches': 59}, {'person.person_id': 'VeraJourova.1964', 'persName.forename': 'Věra', 'persName.surname': 'Jourová', 'total_speeches': 2}]

Running time 6.149658203125 ms

+------------------+-------------------+------------------+----------------+
| person.person_id | persName.forename | persName.surname | total_speeches |
+------------------+-------------------+------------------+----------------+
| AndrejBabis.1954 |       Andrej      |      Babiš       |       59       |
| VeraJourova.1964 |        Věra       |     Jourová      |       2        |
+------------------+-------------------+------------------+----------------+
